--- a/make.sh
+++ b/make.sh
@@ -5,15 +5,14 @@
 
 #alt_cannadic="true"
 #edict="true"
-jawiki="true"
-neologd="true"
-personal_names="true"
-place_names="true"
+#jawiki="true"
+#neologd="true"
+#personal_names="true"
+#place_names="true"
 #skk_jisyo="true"
 #sudachidict="true"
 
 rm -f mozcdic-ut.txt
-rm -rf mozcdic-ut-*/{*,.git}
 
 if [[ $alt_cannadic = "true" ]]; then
 git clone --depth 1 https://github.com/utuhiro78/mozcdic-ut-alt-cannadic.git
